### PR TITLE
chore: add parameterized query definition

### PR DESCRIPTION
**Pull request by yquansah**
Adding the parameterized query definition to the query path on swagger

### CODE REVIEW

**Comment by glinton** [comment]
run `make generate` and push again and this should pass

File: src/common/schemas/Query.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@
       type: string
       enum:
         - flux
+    params:
+      type: object
+      description: >
+          Enumeration of key/value pairs that respresent parameters to be
+          injected into query (can only specify either this field or extern and not both)
```
[the remark this answers is not in the document]
Comment (russorat) [reply src/common/schemas/Query.yml:20]: https://github.com/influxdata/ui/issues/831#issuecomment-797770744

**Review by glinton — APPROVED** [review]

File: src/common/schemas/Query.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (russorat) [reply src/common/schemas/Query.yml:20]: https://github.com/influxdata/idpe/issues/10655

File: contracts/common.yml [hunk 2]
```diff
@@ -5961,6 +5961,10 @@ components:
           type: string
           enum:
             - flux
+        params:
+          type: object
```
Comment (bednar) [inline contracts/common.yml:5965]: @glinton, @yquansah 

Why the `params` is not defined as a dictionary? Something like:

```yaml
params:
  type: object
  additionalProperties: true
  description: |
    Enumeration of key/value pairs that respresent parameters to be injected into query (can only specify either this field or extern and not both)
```

- https://swagger.io/docs/specification/data-models/dictionaries/

**Pull request merged**